test/mainnet/basicfunctionality.spec.sol

```diff
@@ -400,8 +405,8 @@ contract VaultBasicFunctionalityTest is BaseIntegrationTest, TestHelper {
             "Withdrawer shares balance should remain unchanged"
         );
 
-        assertApproxEqAbs(
-            withdrawer.totalAssets(), withdrawerTotalBefore, 3, "Withdrawer total assets should remain unchanged"
+        assertApproxEqRel(
+            withdrawer.totalAssets(), withdrawerTotalBefore, 5e14, "Withdrawer total assets should remain unchanged"
```

Comment:
let's not merge these kind of changes  please, increases deltas here from 3 wei to 0.005 is not justified

Reply:
i'm creating a PR against eth-max-vault-release-candidate